torch/csrc/jit/codegen/cuda/python_frontend/fusion_record.h

```diff
@@ -895,6 +896,41 @@ struct TensorRecord : RecordFunctor {
   bool is_cpu_;
 };
 
+struct NullTensorRecord : RecordFunctor {
+  NullTensorRecord(std::vector<State> _outputs)
+      : RecordFunctor(
+            {},
+            std::move(_outputs),
+            "null_tensor",
+            RecordType::Tensor) {}
```

Comment:
I would add a new Op Type since you have a unique Record like `RecordType::NullTensor`.